client/my-sites/customer-home/controller.jsx

```diff
@@ -13,9 +13,10 @@ import { getSelectedSiteSlug, getSelectedSiteId } from 'state/ui/selectors';
 import { canCurrentUserUseCustomerHome } from 'state/sites/selectors';
 import isRecentlyMigratedSite from 'state/selectors/is-site-recently-migrated';
 
-export default function ( context, next ) {
-	const state = context.store.getState();
-	const siteId = getSelectedSiteId( state );
+export default async function ( context, next ) {
```

Comment:
Do we still need this async change?

Reply:
Oh nm I suppose this one is more around timing

Reply:
Yes, exactly. It was more important when flipping between the two card states, but I think it's fine here for now.